embassy-stm32/src/rcc/g4.rs

```diff
@@ -357,5 +357,8 @@ pub(crate) unsafe fn init(config: Config) {
         adc: adc12_ck,
         adc34: adc345_ck,
         rtc,
+        sysclk: None,
+        noclk: None,
```

Comment:
"noclk" is not a clock, i'd suggest treating it specially in the match, just panic.

Reply:
sorry, I should have marked this as draft. this change was not going to be merged